pkg/operator/ceph/file/mirror/spec_test.go

```diff
@@ -88,6 +90,8 @@ func TestPodSpec(t *testing.T) {
 	assert.Equal(t, 1, len(d.Spec.Template.Spec.Volumes[0].Projected.Sources))
 	assert.Equal(t, 4, len(d.Spec.Template.Spec.Containers[0].VolumeMounts))
 
+	assert.Equal(t, cephcontroller.DefaultServiceAccount, d.Spec.Template.Spec.ServiceAccountName)
```

Comment:
the extra line above.

Reply:
Already did the nfs changes and updated the mirror.